docs-new/docs/contribute/docs/contrib-guidelines-docs.md

```diff
@@ -38,63 +39,55 @@ that are relevant only to documentation
-    for the same reason.
+    For usability considerations, we have a few exceptions to this rule
+    for the main `README.md` file and the [Getting Started Guide](../../getting-started/index.md).
 
 * When you want to display a sample file that exists in the repository,
-  use the `embed path` shortcode syntax
+  use the `include <file-path>` shortcode syntax
   (which automatically pulls the current version of the file into your document)
   rather than copying the text.
   This ensures that, when the sample file is updated,
   your document is also updated.
 
-  For example, to embed the `examples/sample-app/version3/app-pre-deploy-eval.yaml` file,
-  the syntax is:
-
-  ```md
-  {%/*include "examples/sample-app/version-3/app-pre-deploy-eval.yaml"*/%}
-
-  ```
-
-* `markdownlint` enforces limits on line length.
-  Links to other documents are exempted from this limit
-  but, if a line has words before and after the long string,
-  `markdownlint` fails.
-  A good practice is to just code all links on their own page.
-  So, instead of coding:
-
-```shell
-The [Other section](long-link-to-section) page
-...
-```
-
-  you should code the following,
-  unless the link is so short
-  that you are sure it will not violate the line-length rules::
-
-```shell
-The
-[Other section](long-link-to-section)
-page
-...
-```
-
-* Always build the documentation locally to check the formatting
-  and verify that all links are working properly.
-  See [Build Documentation Locally](./local-building.md)
-  for details.
-
-* Always run the following to fix most markdown issues in your PR
-  and identify issues that can not be fixed automatically:
-
-```shell
-make markdownlint-fix
-```
-
-  See [Markdownlint](./linter-requirements.md#markdownlint)
-  for details.
+    For example, to include the `examples/sample-app/version3/app-pre-deploy-eval.yaml` file,
+    the syntax is:
+
+    ```md
+    {% /* include "examples/sample-app/version-3/app-pre-deploy-eval.yaml" */ %}
+    ```
+
+  * `markdownlint` enforces limits on line length.
+    Links to other documents are exempted from this limit
+    but, if a line has words before and after the long string,
+    `markdownlint` fails.
+    A good practice is to just put all links on their own line.
+    So, instead of coding:
+  
+    ```md
+    The [Other section](long-link-to-section) page
+    ```
+
+    you should code the following,
+    unless the link is so short
+    that you are sure it will not violate the line-length rules:
+
+    ```md
+    The
+    [Other section](long-link-to-section)
+    page
+    ```
+
+  * Always build the documentation locally to check the formatting
+    and verify that all links are working properly.
+    See [Build Documentation Locally](./local-building.md)
+    for details.
+
+  * Always run the following to fix most markdown issues in your PR
+    and identify issues that can not be fixed automatically:
+
+    ```shell
+    make markdownlint-fix
```

Comment:
what is the decision with markdown lint? Do we keep it to v 0.11.0 to make this command work? The alternative is to setup linting rules and this does not sound fun

Reply:
will be fixed in #2817

Reply:
No description for #2817.  Will that make `markdownlint-fix` work?  Preserving that `fix` functionality really is critical, IMHO.

Reply:
yes, it will fix the command